tst/standard/display.tst

```diff
@@ -106,6 +106,42 @@ gap> dot{[1 .. 50]};
 #Error, Digraphs: Splash: usage,
 #the option <engine> must be "dot", "neato", "twopi", "circo", "fdp", "sfdp", or "patchwork"
 
+# DotPartialOrderDigraph
+gap> gr := Digraph([[1], [1, 2], [1, 3], [1, 4], [1 .. 5], [1 .. 6],
+> [1, 2, 3, 4, 5, 7], [1, 8]]);;
+gap> dot := DotPartialOrderDigraph(gr);;
```

Comment:
For the test you can just do `Print(DotPartialOrderDigraph(gr);`: it should make much nicer output to copy and paste into a test file. Same in the other places.